tests/test.rs

```diff
@@ -398,6 +398,6 @@ fn bytes() {
     let data = b"abc\0123";
     let s = serialize(&data, Infinite).unwrap();
     let s2 = serialize(&Bytes::new(data), Infinite).unwrap();
-    assert_eq!(s, s2);
+    assert_ne!(s, s2);
 }
 
```

Comment:
Since we do tests for fixed sized arrays, is adding a test for both serialization and deserialization really necessary?